src/index.ts

```diff
@@ -49,6 +48,7 @@ function manageServers(): void {
 }
 
 async function server(): Promise<void> {
+  const { allPlugins } = await import('./plugins/allPlugins')
```

Comment:
This works, but we could keep using normal `import` if we instead export a `makePlugins` factory instead of a baked-out `allPlugins`. Not really a big deal, but I thought I'd mention it in case older node versions have trouble.

Reply:
Well, this isn't top-level await, so there wouldn't be an issue there. However, a factory would be a good alternative as a later followup if this is problematic.